src/GslibIO.jl

```diff
@@ -155,4 +198,52 @@ function save(file::File{format"GSLIB"}, sdata::AbstractData)
        propnames=vars)
 end
 
+# low level function for saving data to a legacy GSLIB format
+function save_legacy(filename::AbstractString, data::AbstractMatrix,
+                     varnames::NTuple, header::AbstractString, na)
+  @assert size(data, 2) == length(varnames) "Invalid data for the specified variable names"
```

Comment:
```suggestion
  @assert size(data, 2) == length(varnames) "invalid data for the specified variable names"
```

Reply:
Just to confirm that assertion messages should not start with upper case. Am I right? To check if this rule is applied in all my changes.

Reply:
yes, all assertions in the code are lowercase and are short sentences